yb-voyager/cmd/migrationStatus.go

```diff
@@ -30,6 +30,14 @@ func GetMigrationStatusRecord() (*MigrationStatusRecord, error) {
 }
 
 func InitMigrationStatusRecord(migUUID string) error {
+	record, err := GetMigrationStatusRecord()
+	if err != nil {
+		return err
+	}
+	if record.MigrationUUID != "" {
+		return nil
+	}
+
```

Comment:
No need of explicitly reading the record here. The `UpdateMigrationStatusRecord()` internally reads the latest record and passes it to the updateFn (the anonymous func). So all you have to do is:
```
	return UpdateMigrationStatusRecord(func(record *MigrationStatusRecord) {
        if record.MigrationUUID != uuid.Nil {
            log.Info("MigrationStatusRecord is already initialised")
            return
        }
		record.MigrationUUID = migUUID
		record.ExportType = SNAPSHOT_ONLY
    })
```